ISI Web of Knowledge.js

```diff
@@ -2,189 +2,14 @@
 	"translatorID": "594ebe3c-90a0-4830-83bc-9502825a6810",
 	"label": "ISI Web of Knowledge",
```

Comment:
"Web of Science Tagged" -- we don't use "Format" for Ovid, either (though we do for RefWorks). That seems OK lengthwise.